lib/generators/shopify_app/app_proxy_controller/app_proxy_controller_generator.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails/generators/base'
+
+module ShopifyApp
+  module Generators
+    class AppProxyControllerGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      def create_app_proxy_controller
+        template 'app_proxy_controller.rb', 'app/controllers/app_proxy_controller.rb'
+      end
+
+      def create_app_proxy_index_view
+        copy_file 'index.html.erb', 'app/views/app_proxy/index.html.erb'
+      end
+
+      def add_app_proxy_route
+        inject_into_file(
```

Comment:
Thanks for the comments, Michael
I did consider using `route` initially over `inject_into_file` but I wasn't sure if route method could inject multiple lines ('\n') at the desired position ( `after: mount...` )